plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/KuCoinConnector.java

```diff
@@ -20,7 +20,7 @@
 
 @FieldDefaults(makeFinal = true, level = PRIVATE)
 public class KuCoinConnector implements IConnector {
-    private static final String ID = EveryTradePlugin.ID + IPlugin.PLUGIN_PATH_SEPARATOR + "kuCoinApiConnector";
+    private static final String ID = EveryTradePlugin.ID + IPlugin.PLUGIN_PATH_SEPARATOR + "kucoinApiConnector";
```

Comment:
I believe that this will break any existing  connector. Is kucoin in production?

Reply:
this will be fixed in db